Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/onboarding/OnboardingNotificationsFragment.kt

```diff
@@ -31,15 +47,15 @@ class OnboardingNotificationsFragment : Fragment(R.layout.fragment_onboarding_no
     }
 
     private fun setButtonOnClickListener() {
-        binding.onboardingButtonNext.setOnClickListener {
-            navigateToMain()
-        }
-        binding.onboardingButtonBack.buttonIcon.setOnClickListener {
-            (activity as OnboardingActivity).goBack()
-        }
+        binding.onboardingButtonNext.setOnClickListener { vm.onNextButtonClick() }
```

Comment:
Inline this please.
If you want to extract something, extract it "by feature", i.e. do 
`setSetupOnboarding` which does buttons + observation.

But don't do `setAllButtons`, we want "package by feature".